dbt_project.yml

```diff
@@ -32,6 +37,7 @@ clean-targets:         # directories to be removed by `dbt clean`
 # files using the `{{ config(...) }}` macro.
 models:
   dbt_arghyam:
+    +quote_columns: true
     pre-hook:
```

Comment:
what is this for ? since we just introduced it.

Reply:
At the beginning, dbt test was not accepting hindi names and was returning a syntax error (hindi being non-ASCII) . I was trying to apply all suggested solutions by chatgpt to make it work, this was one of the suggestion